app/scripts/reducers/granule-csv.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+import {
+  GRANULE_CSV,
+  GRANULE_CSV_INFLIGHT,
+  GRANULE_CSV_ERROR
+} from '../actions/types';
+
+export const initialState = {
+  list: {
```

Comment:
Reading your reducer below, it seems like the only properties you assign to the state are `data`, `inflight`, and `error`:

`{ data: action.data, inflight: false, error: null }`

So it seems like the rest of the properties on the initialState here aren't necessary? Also, you may want to make sure that you have initial values for the properties that you are using